File: coresuite-asp-dotnet-core-mvc-cs/Models/EnvelopeModel.cs
namespace coresuite_asp_dotnet_core_mvc_cs.Models
{
    public class EnvelopeModel
    {
        private Address fromAddress = null;
        private Address toAddress = null;

        public Address FromAddress
        {
            get
            {
                if (fromAddress == null)
                {
                    fromAddress = new Address() { Address1 = "13071 Wainwright Road", Address2 = "Suite 100", City = "Columbia", Name = "", State = "MD", Zip = "20777" };
                }
                return fromAddress;
            }

            set => fromAddress = value;
        }

        public Address ToAddress
        {
            get
            {
                if (toAddress == null)
                {
                    toAddress = new Address() { Address1 = "123 Main Street", Address2 = "", City = "Anywhere", Name = "Any Company", State = "MD", Zip = "20815-4704" };
                }
                return fromAddress;
            }

            set => toAddress = value;
        }
    }
}

File: coresuite-asp-dotnet-framework-mvc-vb/Models/EnvelopeModel.vb
Namespace Models

    Public Class EnvelopeModel

        Private fromAddress1 As Address = Nothing
        Private toAddress1 As Address = Nothing

        Public Property FromAddress As Address

            Get

                If fromAddress1 Is Nothing Then

                    fromAddress1 = New Address() With {.Address1 = "13071 Wainwright Road", .Address2 = "Suite 100", .City = "Columbia", .Name = "", .State = "MD", .Zip = "20777"}
                End If

                Return fromAddress1
            End Get

            Set(ByVal value As Address)

                fromAddress1 = value
            End Set

        End Property

        Public Property ToAddress As Address

            Get

                If toAddress1 Is Nothing Then

                    toAddress1 = New Address() With {.Address1 = "123 Main Street", .Address2 = "", .City = "Anywhere", .Name = "Any Company", .State = "MD", .Zip = "20815-4704"}
                End If
                Return fromAddress1
            End Get

            Set(ByVal value As Address)

                toAddress1 = value
            End Set
        End Property
    End Class
End Namespace
